Quadriga/src/main/java/edu/asu/spring/quadriga/domain/workspace/ITextFile.java

```diff
@@ -29,5 +29,16 @@
     public abstract String getFileContent();
 
     public abstract void setFileContent(String fileContent);
+    
+    public abstract String getTextFileURI();
+
+    /**
+     * @param uriPrefix
+     *      Prefix of the URI specified in the properties file.
+     *      eg: http://quadriga.io/text/textid
+     * 
+     * Sets the URI corresponding to the TextFile. 
+     */
```

Comment:
Read this:
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format
and this:
http://www.monperrus.net/martin/how-to-write-good-API-documentation

And then rewrite your javadoc accordingly.